semver.md

```diff
@@ -6,7 +6,7 @@ Summary
 
 Given a version number MAJOR.MINOR.PATCH, increment the:
 
-1. MAJOR version when you make incompatible API changes,
+1. MAJOR version when you make incompatible changes,
```

Comment:
Rather than remove API, why not also include package?  I think that in the case of NPM and other package managers, the defined dependency tree is part of the package API, but that might be stretching it a bit.

... when you make incompatible package or API changes,

Reply:
```
1. MAJOR version when you make incompatible package or API changes,
```
I like how it sounds.